metric_learn/oasis.py

```diff
@@ -0,0 +1,24 @@
+from .base_metric import BilinearMixin
+import numpy as np
+
+
+class OASIS(BilinearMixin):
```

Comment:
for now you could move this to the test file and define a `DummyBilinearMixin` class there for testing purposes

Reply:
I just moved all to a testing folder and created a `DummyBilinearMixin` for testing purposes